articles/storage/blobs/data-lake-storage-access-control.md

```diff
@@ -46,7 +46,8 @@ To set file and directory level permissions, see any of the following articles:
 |REST API |[Path - Update](/rest/api/storageservices/datalakestoragegen2/path/update)|
 
 > [!IMPORTANT]
-> If the security principal is a *service* principal, it's important to use the object ID of the service principal and not the object ID of the related app registration. To get the object ID of the service principal open the Azure CLI, and then use this command: `az ad sp show --id <Your App ID> --query objectId`. make sure to replace the `<Your App ID>` placeholder with the App ID of your app registration.
+> If the security principal is a *service* principal, it's important to use the object ID of the service principal and not the object ID of the related app registration. To get the object ID of the service principal open the Azure CLI, and then use this command: `az ad sp show --id <Your App ID> --query objectId`. make sure to replace the `<Your App ID>` placeholder with the App ID of your app registration. 
```

Comment:
It looks like you just want to add this line to the existing note "Here the service principal identities referred as users." First off, I'd remove the "++". Seems like that might have been added to this text in error. Also, I think I know what you are trying to convey here. The article has not yet introduced the term "user" yet so it might be a bit early connect a service principal object ID to the user entry of an ACL. If you want to make that connection this early in the article, then perhaps a bit more text would be helpful. Perhaps something like "The service principal is treated as a named user. You'll add this ID to the ACL as you would any named user. Named users are described later in this article". Maybe something like that?